src/libs/actions/ReportActions.ts

```diff
@@ -24,6 +25,10 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction) {
             Onyx.set(`${ONYXKEYS.COLLECTION.TRANSACTION}${linkedTransactionID}`, null);
         }
 
+        // Delete the failed task report too
+        if (reportAction.childReportID) {
+            Report.deleteReport(reportAction.childReportID);
+        }
```

Comment:
Is the task listed under as the childReportID / it should be listed as the taskReportID the message as well. Can we use that so this doesn't create some unintended use cases for other future flows?